tests/Domain/InvoiceItemsTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DDDWorkshop\Domain;
+
+use DDDWorkshop\Domain\ValueObjects\InvoiceItem;
+use DDDWorkshop\Domain\ValueObjects\InvoiceItems;
+use PHPUnit\Framework\TestCase;
+
+class InvoiceItemsTest extends TestCase
```

Comment:
you can also test immutability - that after adding an item, old items still return the same total amount and new items return a new total amount